src/commands/endpoints/access/add.ts

```diff
@@ -18,18 +18,19 @@ export default class EndpointsAccessAdd extends BaseCommand {
 
   static examples = [
     '$ heroku endpoints:access:add postgresql-rigid-37567 --account_ids 123456',
-    '$ heroku endpoints:access:add postgresql-rigid-37567 --account_ids 123456,78910',
+    '$ heroku endpoints:access:add postgresql-rigid-37567 --account_ids 123456 78910',
   ]
 
   async run() {
     const {args, flags} = this.parse(EndpointsAccessAdd)
     const database = args.database || await fetcher(this.heroku, flags.app)
-
-    cli.action.start('Adding account to the whitelist')
+    const account_ids = flags['account-ids'].split(' ').map((account: any) => account.trim())
```

Comment:
Having `accountIds` is more consistent, no?

Reply:
I think CLI team prefers kebab case

Reply:
Sorry I wasn't clear: I meant for the variable name, instead of `account_ids`.